DigitalLearningSolutions.Web.Tests/Helpers/SelectListHelperTests.cs

```diff
@@ -95,5 +96,27 @@ public void GetOptionsWithSelectedValue_with_list_of_strings_returns_populated_l
                 result.ElementAt(1).Selected.Should().BeFalse();
             }
         }
+
+        [Test]
+        public void MapPeriodOptionsToSelectListItem_returns_expected_list()
+        {
+            // When
+            var result = SelectListHelper.MapPeriodOptionsToSelectListItem(14);

```

Comment:
The name of this method feels wrong to me as "map" should map one list to another, but we're only passing in a value that then becomes a list.